crates/meroctl/src/cli/app.rs

```diff
@@ -19,13 +21,15 @@ pub enum AppSubCommands {
     Install(InstallCommand),
     #[command(alias = "ls")]
     List(ListCommand),
+    Get(GetCommand),
 }
 
 impl AppCommand {
     pub async fn run(self, args: RootArgs) -> EyreResult<()> {
         match self.subcommand {
             AppSubCommands::Install(install) => install.run(args).await,
             AppSubCommands::List(list) => list.run(args).await,
+            AppSubCommands::Get(get) => get.run(args).await,
```

Comment:
Nit: If we keep these things in alphabetical order as a general rule it helps overall and over time as things grow 🙂